admin_manual/configuration_files/files_locking_enabling.rst

```diff
@@ -16,4 +17,28 @@ underlying filesystem.
 The Files Locking app has no configuration options; all you need to do is 
 enable or disable it on your Apps page.
 
-.. figure:: ../images/files_locking_app.png
\ No newline at end of file
+.. figure:: ../images/files_locking_app.png
+
+We recommend using `Redis <http://redis.io/>`_ as your ownCloud memcache when 
+you enable the Files Locking app. Memcached, the popular distributed memory 
```

Comment:
> if the new file locking mechanism works well, it will obsolete the old files_locking app.

Really? I thought that files_locking is for locking against concurrent processes to the ownCloud server on the server side (filesystem level).